code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -322,6 +322,9 @@
 	if(method == REAGENT_TOUCH)
 		heal_overall_damage(M, volume)
 
+	if(method == REAGENT_INGEST)
+		data = "Ingested" // sin
```

Comment:
You might want to add a column to data, so it doesn't fully override the other data, like blood types.

Reply:
agreed